diff --git a/node_modules/@types/chrome/index.d.ts b/node_modules/@types/chrome/index.d.ts
index aef9202..64523d2 100644
--- a/node_modules/@types/chrome/index.d.ts
+++ b/node_modules/@types/chrome/index.d.ts
@@ -9694,6 +9694,13 @@ declare namespace chrome.tabs {
      * @since Chrome 16.
      */
     export function reload(callback?: () => void): void;
+    /**
+     * Duplicates a tab.
+     * @since Chrome 23.
+     * @param tabId The ID of the tab which is to be duplicated.
+     * @return The `duplicate` method provides its result via callback or returned as a `Promise` (MV3 only).
+     */
+    export function duplicate(tabId: number): Promise<Tab|undefined>;
     /**
      * Duplicates a tab.
      * @since Chrome 23.
